pipeline/runtime/runtestsV2.go

```diff
@@ -270,14 +269,14 @@ func getPreCmd(workspace, tmpFilePath string, fs filesystem.FileSystem, log *log
 		return "", "", err
 	}
 
-	bazelfilepath, err := writetoBazelrcFile(log, fs, tmpFilePath, splitIdx)
+	_, err = writetoBazelrcFile(log, fs, tmpFilePath, splitIdx)
```

Comment:
We can change the method signature of returning type error only if we are not using string return type anywhere.